setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
Also you can override any of these by putting a comment on that line with `noqa`